tests/gradients/finite_diff/test_finite_difference_shot_vec.py

```diff
@@ -706,15 +704,16 @@ def test_probs_expval_with_argnum_one(self, approx_order, strategy, validate):
 
         cx, sx, cy, sy = np.cos(x / 2), np.sin(x / 2), np.cos(y / 2), np.sin(y / 2)
         # probability vector is [cx**2 * cy**2, cx**2 * sy**2, sx**2 * sy**2, sx**2 * cy**2]
-        exp_dprob = np.array([-(cx**2), cx**2, sx**2, -(sx**2)]) * (2 * sy * cy)
+        exp_dprob = np.array([-(cx**2), cx**2, sx**2, -(sx**2)]) * (sy * cy)
         exp_probs = [np.zeros(4), exp_dprob]
-        exp_expval = [0, -np.sin(y) * np.cos(x)]
+        # expval is np.sin(y) * np.cos(x)
+        exp_expval = [0, np.cos(y) * np.cos(x)]
```

Comment:
is this value the derivative then? should it be `np.cos(y) * -np.sin(x)`? guessing not since it's way different

Reply:
Looking at the probability derivatives, I think we're only considering derivatives wrt `y`